src/libraries/System.Net.HttpListener/src/System/Net/Windows/WebSockets/WebSocketBase.cs

```diff
@@ -1020,23 +1019,16 @@ private void ReleaseLocks(ref bool thisLockTaken, ref bool sessionHandleLockTake
 
             if (thisLockTaken || sessionHandleLockTaken)
```

Comment:
This `if` is no longer necessary and can just be removed (the body should remain).